src/Sarif/Visitors/InsertOptionalDataVisitor.cs

```diff
@@ -361,6 +393,20 @@ private string GetNextRepoRootUriBaseId()
             }
         }
 
+        private string GetFilePath(ArtifactLocation node)
+        {
+            Uri uri = node.Uri;
```

Comment:
>Uri uri = node.Uri [](http://example.com/codeflow?start=12&length=18)

check ArtifactLocation.Resolve, that might simplify a little bit

Reply:
Fixed.